oob/generalities/shorthands.tex

```diff
@@ -60,11 +63,11 @@
 \saNote{} 
 If the \textsc{Ethereum} network configuration is at the \textsc{Cancun} hard fork, one \emph{may} further impose
 \[
-	\locFlagSumNewPraguePrecompiles _{i} = 0 \qquad (\sanityCheck)
+	\locFlagSumNewPraguePrecompiles _{i} + \locFlagSumNewOsakaPrecompiles _{i} = 0 \qquad (\sanityCheck)
 \]
-thereby preventing \textsc{Prague}-specific \texttt{BLS} precompiles to be executed.
-This isn't strictly speaking necessary, as the \trmMod{} won't recognize addresses in the range
-$[\texttt{0x\,0b}..\texttt{0x\,11}]$
+thereby preventing \textsc{Prague}-specific \texttt{BLS} and \textsc{Osaka}-specific precompiles to be executed.
+This isn't strictly speaking necessary, as the \trmMod{} won't recognize addresses in the set
+$[\texttt{0x\,0b}..\texttt{0x\,11}] \cup [\texttt{0x\,100}]$
```

Comment:
If it isn't already present I would introduce `\locPrcFlagSumCancun`, `\locPrcFlagSumPrague` and `\locPrcFlagSumOsaka` (with `\locPrcFlagSumBls = \locPrcFlagSumCancun + \locPrcFlagSumPrague`) and re-write this paragraph to say that if the network is running the XXX-EVM then `\locPrcFlagSumYYY` should be 0 for any fork YYY that is beyond fork XXX. This way the paragraph won't have to be changed in the future (and fixes the missing text about Osaka precompiles also not being active on Prague).